### PR TITLE
[Ready] Upgrade to Spaceship-ITC

### PR DESCRIPTION
**Pull request by mathcarignani**
This pull request will upgrade `Produce` to use the new `Spaceship` ITC feature

### CODE REVIEW

File: lib/produce/itunes_connect.rb [hunk 1]
```diff
@@ -1,32 +1,13 @@
 require 'fastlane_core/itunes_connect/itunes_connect'
```
Comment (KrauseFx) [inline lib/produce/itunes_connect.rb]: We can drop `ItunesConnect` completely :smile:

Comment (mathcarignani) [inline lib/produce/itunes_connect.rb]: Definitely :+1:

File: lib/produce/options.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        FastlaneCore::ConfigItem.new(key: :primary_language,
+                                     env_name: "PRODUCE_LANGUAGE",
+                                     description: "Primary Language (e.g. 'English', 'German')",
+                                     verify_block: Proc.new do |language|
+                                       raise "Please enter one of available languages: #{AvailableDefaultLanguages.all_languages}".red unless AvailableDefaultLanguages.all_languages.include?(language)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :skip_itc,
+                                     short_option: "-i",
+                                     env_name: "PRODUCE_SKIP_ITC",
+                                     description: "Skip ITC",
```
Comment (KrauseFx) [inline lib/produce/options.rb]: Please change to `Skip the creation of the app on iTunes Connect`

File: lib/produce/options.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                                     is_string: false,
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :skip_devcenter,
+                                     short_option: "-d",
+                                     env_name: "PRODUCE_SKIP_DEVCENTER",
+                                     description: "Skip Developer Center",
```
Comment (KrauseFx) [inline lib/produce/options.rb]: Please change to `Skip the creation of the app on the Apple Developer Portal`

File: lib/produce/options.rb (hunk 2 above)
Comment (mathcarignani) [inline lib/produce/options.rb]: :+1:

File: lib/produce/options.rb (hunk 3 above)
Comment (mathcarignani) [inline lib/produce/options.rb]: :+1:

**Comment by KrauseFx** [comment]
Is it ready? If so, you can update the title to `[Ready]`

**Comment by mathcarignani** [comment]
Yes is ready ;) I want to test it again once the PR in `spaceship` is merged

**Comment by KrauseFx** [comment]
Oh, right, the release will wait until the `spaceship` for iTunes Connect was released.

**Comment by KrauseFx** [comment]
Thanks again @mathcarignani :+1:

**Pull request merged**